pkg/envoy/lds/listener.go

```diff
@@ -35,13 +35,9 @@ func (lb *listenerBuilder) newOutboundListener() (*xds_listener.Listener, error)
 		Address:          envoy.GetAddress(constants.WildcardIPAddr, constants.EnvoyOutboundListenerPort),
 		TrafficDirection: xds_core.TrafficDirection_OUTBOUND,
 		FilterChains:     serviceFilterChains,
-		ListenerFilters: []*xds_listener.ListenerFilter{
-			{
-				// The OriginalDestination ListenerFilter is used to redirect traffic
-				// to its original destination.
-				Name: wellknown.OriginalDestination,
```

Comment:
Let's confirm whether the OriginalDestination filter is required for in-mesh connectivity on Windows. +@davinci26